documentation/Maintenance.md

```diff
@@ -833,7 +833,8 @@ nodes:
 * If you intend to add the new `balancer` node with VRRP IP, and have previously not configured the `vrrp_ips` section, you need to do the following preliminarily:
   * And the section to the main `cluster.yaml`.
   * If you already have balancers without VRRP IPs, reconfigure the balancers and DNS,
-    for example, using `kubemarine install --tasks prepare.dns.etc_hosts,deploy.loadbalancer.haproxy.configure,deploy.loadbalancer.keepalived,deploy.coredns`
+    for example, using `kubemarine install --tasks prepare.dns.etc_hosts,deploy.loadbalancer.haproxy.configure,deploy.loadbalancer.keepalived,deploy.coredns`.
+* If you intend to add the new `balancer` node with VRRP IP, and have previously configured the `vrrp_ips` section in the `cluster.yaml`, you have to add the adding node to the `vrrp_ips` section in the `cluster.yaml` the same way as the old balancer nodes.
```

Comment:
Yes, in case of no hosts in vrrp_ips, they will be updated with all nodes with `balancer` role. But if there already are some hosts in vrrp_ips, then this `if` is not executed.

And looks like the issue is here: https://github.com/Netcracker/KubeMarine/blob/9da716bec6aafba4b3c78e007f58421a61450d88/kubemarine/keepalived.py#L146-L159

If a node is not in vrrp_ips' hosts, it will not be updated with `keepalived` role and keepalived will not be installed there.

Reply:
Ok, in this case the statement is not clear enough. It is a quite unusual configuration with specific hosts. Most of the users have vrrp_ips section configured in the `cluster.yaml` but should not have the described problem, and do not have to add the adding node to the `vrrp_ips` section. Maybe we can clarify exact case with examples.

Reply:
More detailed explanation of the case is added as well as an example.